buttons/src/main/res/values/attrs.xml

```diff
@@ -15,6 +15,24 @@
     <attr name="vtmnButtonGhostReversedMediumStyle" format="reference"/>
     <attr name="vtmnButtonGhostReversedLargeStyle" format="reference"/>
 
+    <attr name="vtmnIconButtonPrimaryMediumStyle" format="reference"/>
+    <attr name="vtmnIconButtonPrimaryLargeStyle" format="reference"/>
+    <attr name="vtmnIconButtonSecondaryMediumStyle" format="reference"/>
+    <attr name="vtmnIconButtonSecondaryLargeStyle" format="reference"/>
+    <attr name="vtmnIconButtonTertiaryMediumStyle" format="reference"/>
+    <attr name="vtmnIconButtonTertiaryLargeStyle" format="reference"/>
+    <attr name="vtmnIconButtonConversionMediumStyle" format="reference"/>
+    <attr name="vtmnIconButtonConversionLargeStyle" format="reference"/>
+    <attr name="vtmnIconButtonPrimaryReversedMediumStyle" format="reference"/>
+    <attr name="vtmnIconButtonPrimaryReversedLargeStyle" format="reference"/>
+    <attr name="vtmnIconButtonGhostMediumStyle" format="reference"/>
+    <attr name="vtmnIconButtonGhostLargeStyle" format="reference"/>
+    <attr name="vtmnIconButtonGhostReversedMediumStyle" format="reference"/>
+    <attr name="vtmnIconButtonGhostReversedLargeStyle" format="reference"/>
+
+    <attr name="vtmnIconButtonMediumSize" format="dimension" />
```

Comment:
Why declare this as an attr? Is it not working with just a simple dimens?

Reply:
You can customize it without attr. Simply write a dimens with the same name in your project will override it

Reply:
Yes but you need to know the name of this dimen which is internal. I don't think it is the best way.